packages/blitz/src/paginate.ts

```diff
@@ -48,11 +48,15 @@ export async function paginate<QueryResult>({
 
   const hasMore = skip + take < count
   const nextPage = hasMore ? {take, skip: skip + take} : null
+  const pageCount = Math.floor((count + take - 1) / take)
+  const from = skip + 1
+  const to = skip + take
 
   return {
     items,
     nextPage,
     hasMore,
+    pagination: {pageCount: pageCount, pageSize: take, totalCount: count, from, to},
```

Comment:
```suggestion
    pageCount,
    pageSize,
    from,
    to,
```

Reply:
Hey @paulm17, seems your latest commit still doesn't address this comment

Reply:
Did commit [1788cd3](https://github.com/blitz-js/blitz/pull/3970/commits/1788cd378a92c402423d16d62ecbb42409eeb6ce) do the required change?